device-catalog.html

```diff
@@ -9,186 +9,224 @@
-                        <input type="range" name="price" id="filter-price-from" min="0" step="500" value="5000">
-                        <input type="range" name="price" id="filter-price-to" max="10000" step="500" value="5000">
-                        <!-- С этим у меня пока проблема -->
-                    </fieldset>
-                    <fieldset>
-                        <legend>Цвет</legend>
-                        <ul>
-                            <li>
-                                <input type="checkbox" name="black" id="filter-color-black" checked>
-                                <label for="filter-color-black">Черный</label>
-                            </li>
-                            <li>
-                                <input type="checkbox" name="white" id="filter-color-white">
-                                <label for="filter-color-white">Белый</label>
-                            </li>
-                            <li>
-                                <input type="checkbox" name="blue" id="filter-color-blue" checked>
-                                <label for="filter-color-blue">Синий</label>
-                            </li>
-                            <li>
-                                <input type="checkbox" name="red" id="filter-color-red" checked>
-                                <label for="filter-color-red">Красный</label>
-                            </li>
-                            <li>
-                                <input type="checkbox" name="pink" id="filter-color-pink">
-                                <label for="filter-color-pink">Розовый</label>
-                            </li>
-                        </ul>
-                    </fieldset>
-                    <fieldset>
-                        <legend>Bluetooth</legend>
-                        <ul>
-                            <li>
-                                <input type="radio" name="bluetooth" value="on" id="filter-bluetooth-on">
-                                <label for="filter-bluetooth-on">Есть</label>
-                            </li>
-                            <li>
-                                <input type="radio" name="bluetooth" value="off" id="filter-bluetooth-off">
-                                <label for="filter-bluetooth-off">Нет</label>
-                            </li>
-                        </ul>
-                    </fieldset>
-                    <button type="submit">Показать</button>
-                </form>                
-            </aside>
+            <section class="section-filter">
+                <h2 class="visually-hidden">Фильтр товаров</h2>
+                <div class="sorting">
+                    <h2>Сортировка:</h2>
+                    <ul class="sorting-style">
+                        <li>
+                            <label for="price">
+                                <input type="radio" name="content" value="price" id="price" checked>
```

Comment:
too complicated, I would use just links

Reply:
as you can see it was as links. But then....
Changed to the links